app/models/concerns/profile_searching.rb

```diff
@@ -8,10 +8,11 @@ module ProfileSearching
     scoped_search on: %i[id name ref_id account_id compliance_threshold
                          external parent_profile_id]
     scoped_search relation: :assigned_hosts, on: :id, rename: :system_ids
-    scoped_search relation: :assigned_hosts, on: :name, rename: :system_names
+    scoped_search relation: :assigned_hosts, on: :display_name,
+                  rename: :system_names
     scoped_search relation: :test_result_hosts, on: :id,
                   rename: :test_result_system_ids
-    scoped_search relation: :test_result_hosts, on: :name,
+    scoped_search relation: :test_result_hosts, on: :display_name,
                   rename: :test_result_system_names
```

Comment:
This should fix [RHICOMPL-1111](https://issues.redhat.com/browse/RHICOMPL-1111).

Reply:
this is awesome!

Reply:
will this search with `name` and `display_name`?

Reply:
`name` was a field we were saving in our own database. afaik, there is no `name` field in inventory, only `display_name`.